src/otx/algorithms/detection/configs/instance_segmentation/convnext_maskrcnn/compression_config.json

```diff
@@ -20,6 +20,11 @@
           "algorithm": "quantization",
           "initializer": {
             "range": {
+              "type": "percentile",
+              "params": {
+                "min_percentile": 0,
+                "max_percentile": 100
+              },
```

Comment:
Sorry for bothering you. Is this because the update of default configurations from NNCF side?

Reply:
Yes, that's current workaround to address failures after refactoring in NNCF statistic collectors